activemq/datadog_checks/activemq/data/conf.yaml.example

```diff
@@ -13,7 +13,7 @@ instances:
     ## @param port - integer - required
     ## ActiveMQ port to connect to.
     #
-    port: 1099
+    port: 1616
```

Comment:
Do you have any context regarding this? Is 1616 always the standard? Any idea why we had 1099 before?

Reply:
Not sure, but the docs that I read including the ones linked by our README for how to setup JMX all used port 1616.  The docker container also had it exposed by default.

See JMX doc here:
https://activemq.apache.org/jmx.html

Reply:
We seem to have `7199` in our docs https://docs.datadoghq.com/integrations/activemq/#pagetitle 🤦‍♂ 

Yeah let's go with this change, do you mind updating the docs (the ReadMe) as well?

Reply:
Also quick note that activemq has a custom doc page (with both activemq and activemq_xml) on the same page. Updating the ReadMe may not be enough so: cc @DataDog/baklava

Reply:
Interesting, looks like that part of the README was last updated in 2017, so defaults may have changed since then.